composer.json

```diff
@@ -17,15 +17,12 @@
     "require": {
         "php": ">=5.3.9",
         "symfony/framework-bundle": "~2.3|~3.0",
+        "symfony/yaml": "~2.3|~3.0",
+        "symfony/property-access": "~2.3|~3.0",
         "jms/serializer-bundle": "~0.13|~1.0|~2.0",
         "pagerfanta/pagerfanta": "~1.0",
         "willdurand/hateoas": ">2.0"
     },
-    "require-dev": {
-        "symfony/expression-language": "~2.4",
```

Comment:
`expression-language` is not needed here anymore? I definitely use it in my PR :)

Reply:
@RuslanZavacky I didn't found any usage in master branch, that's why I removed it.
In your pull request this library I hope will be added as regular required library, not dev.
Are we good?